src/components/Pages/HomePage/HomePage.css

```diff
@@ -1,3 +1,80 @@
-.div {
+.home-content {
     text-align: center;
-}
\ No newline at end of file
+    position: absolute;
+    width: 100%;
+    height: 660px;
+    left: 0px;
+    top: 67px;
+    background: #F5F9FF;
+    background-size: cover;
+  }
+
+  .home-title{
+    position: absolute;
+    width: 469px;
+    height: 115px;
+    left: 144px;
+    top: 183px;
+
+    font-family: 'Montserrat';
```

Comment:
The reason the font isn't coming through is because you haven't specified a generic family.
Try doing this: `font-family: "Montserrat", sans-serif;`